gnat2goto/driver/tree_walk.adb

```diff
@@ -928,8 +1029,10 @@ package body Tree_Walk is
       end if;
       return R : constant Irep := New_Irep (I_Bounded_Signedbv_Type) do
          Set_Width (R, Get_Width (Resolved_Underlying));
-         Set_Lower_Bound (R, Do_Constant (Low_Bound (Range_Expr)));
-         Set_Upper_Bound (R, Do_Constant (High_Bound (Range_Expr)));
+         Set_Lower_Bound (R, Uint_To_Integer (Intval (Low_Bound
```

Comment:
I still don't think it makes much sense for these to be properties on the type; For dynamic bounds at least we'll need to carry around the bounds in variables/hidden parameters anyway

Reply:
What is keeping us from using unbounded signedbv types for the bounds here by the way?

Reply:
@hannes-steffenhagen-diffblue I'm not sure what is going to be the best approach for dynamic bounds. As of now, the bounds are `Uint`s (as per the last commit) and the `Irep` properties are `Integer`. Having any other numeric type inside `Irep`s would require changing the schemes, and the generating Python script.